modules/entity/spec/sorted_state_adapter.spec.ts

```diff
@@ -20,9 +20,7 @@ describe('Sorted State Adapter', () => {
   });
 
   afterAll(() => {
-    Object.defineProperty(Array.prototype, 'unwantedField', {
-      value: undefined,
-    });
+    delete (Array.prototype as any).unwantedField;
```

Comment:
Had to do this otherwise the integration router specs were failing.
Could the creation of this property on the array be left out?

```
TypeError: Cannot assign to read only property 'unwantedField' of object '[object Array]'
        at new NodeList (C:\Users\tdeschryver\dev\platform\node_modules\domino\lib\NodeList.es6.js:8:44)
        at Document.value (C:\Users\tdeschryver\dev\platform\node_modules\domino\lib\Document.js:726:33)
        at DominoAdapter.BrowserDomAdapter.querySelectorAll (C:\Users\tdeschryver\dev\packages\platform-browser\src\browser\browser_adapter.ts:122:66)
        at DOMTestComponentRenderer.insertRootElement (C:\Users\tdeschryver\packages\platform-browser-dynamic\testing\src\dom_test_component_renderer.ts:25:31)
        at TestBedViewEngine.createComponent (C:\Users\tdeschryver\packages\core\testing\src\test_bed.ts:615:27)
```

Reply:
I think its ok leave it here